ton-connect/Cargo.toml

```
@@ -21,7 +21,11 @@ serde_with = { workspace = true, features = ["chrono_0_4"] }
 tlb-ton = { workspace = true, features = ["serde"] }
```

make it optional and enable when `cell` feature is enabled
```suggestion
tlb-ton = { workspace = true, features = ["serde"], optional = true }
```